### PR TITLE
Addition of `loodb` package

### LINKED ISSUE #337: Pull report creation code into its own interface
The migration code is getting a little cumbersome, as it's mixing logic for the creation new-schema models and the extraction of old-schema models. Pulling out this new-schema creation logic would make the migration code cleaner and more reusable.

### PR DESCRIPTION
**Pull request by Hoolean**
A `loodb` package has been added; definitions and logic related to the new Mongoose schemas are situated here.

Some methods have been added that, as well as being immediately useful for migration code, should be useful in the future. One can imagine that more will be added when `api` is refactored to use the new schemas.

Closes #337.

### CODE REVIEW

File: packages/migrate/bin/summarise-report-values.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ const fs = require('fs');
 
 // the properties that can be in a new report
 const properties = Object.keys(
-  require('../lib/models').Report.schema.tree.diff
+  require('../src/models').Report.schema.tree.diff
```
Comment (ob6160) [inline packages/migrate/bin/summarise-report-values.js]: This line causes an error for me - the file seems to have been moved to the `@neontribe/gbptm-loodb` package.

File: packages/ui/src/pages/AboutPage.js [hunk 2]
```diff
@@ -127,7 +127,8 @@ class AboutPage extends Component {
               </p>
```
Comment (ob6160) [inline packages/ui/src/pages/AboutPage.js]: Was this caused by the linter/formater?

File: packages/migrate/bin/summarise-report-values.js (hunk 1 above)
Comment (Hoolean) [reply packages/migrate/bin/summarise-report-values.js]: Correct; I'll point it in the right place else find a better way to source the property keys.

File: packages/ui/src/pages/AboutPage.js (hunk 2 above)
Comment (Hoolean) [reply packages/ui/src/pages/AboutPage.js]: Yep, didn't intentionally touch anything in `ui`.

**Pull request merged**